devtools/conda-envs/test_env.yaml

```
@@ -6,10 +6,10 @@ dependencies:
   - python
   - pip
   - numpy
-  - MDAnalysis
-  - mdtraj # temporary; remove after refactoring
   - sphinx
-
+  - MDAnalysis==2.0.0-dev0
```

I think you need to install it explicitly via pip:

```suggestion
  - pip:
      - MDAnalysis==2.0.0-dev0
```

---

Because it's relying on the `Results` class, could you raise an issue as a reminder that MDA should get pinned to >=2.0.0 in `setup.py`? (I wouldn't do it now, but it'd be a good thing to do before the first release)

---

Also +1 to @RMeli's comment. The beta needs to be installed via `pip` (swapping `-dev0` with `b0` should do the trick).

---

Btw, @IAlibay, this PR demonstrates to be that the beta was worth the effort because it makes it a lot easier for packages to test against what 2.0 will look like.